DataConnectors/SentinelOne/SentinelOneSentinelConnector/__init__.py

```diff
@@ -151,6 +152,7 @@ def get_installed_apps(self, report_type_suffix, report_type_name, name, params
                 logging.error("Something wrong. Error code: {}".format(r.status_code))
         except Exception as err:
             logging.error("Something wrong. Exception error text: {}".format(err))
+            logging.debug("Status code for installed apps: " + str(r.status_code))

```

Comment:
Use:
logging.debug("Status code for reports: %s" % r.status_code)

Reason:
https://dev.to/izabelakowal/what-is-the-best-string-formatting-technique-for-logging-in-python-d1d